config/prow/cluster/k8sgsmtools-config.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  namespace: k8s-gsm-tools
+  name: config
+data:
+  rotConfig: |
+    specs:
+    - project: spiffxp-gke-dev
+      secret: service-account
```

Comment:
For now, in order to bootstrap, the user needs to create at least an empty gsm secret `projects/spiffxp-gke-dev/secrets/service-account`. We can consider to upsert the gsm secret in the case that 1) the project exists, and 2) the secret does not yet exist.